Jint/Native/JsValue.cs

```diff
@@ -664,7 +666,7 @@ public JsValue Invoke(JsValue thisObj, JsValue[] arguments)
 
         public static bool ReturnOnAbruptCompletion(ref JsValue argument)
         {
-            if (argument.IsCompletion())
+            if (!argument.IsCompletion())
```

Comment:
knockout parsing failed as wrong check was made and non-completion was tried to be casted to completion